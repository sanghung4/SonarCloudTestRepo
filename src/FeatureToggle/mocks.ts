import { Feature } from 'generated/graphql';

export const mockFeatures: Feature[] = [
  { id: '1', isEnabled: false, name: 'noooo' },
  { id: '2', isEnabled: true, name: 'fine' },
  { id: '3', isEnabled: true, name: 'test' }
];
