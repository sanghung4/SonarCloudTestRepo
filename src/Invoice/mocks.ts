import { GetInvoiceDocument } from 'generated/graphql';

export const successMincron = {
  request: {
    query: GetInvoiceDocument,
    variables: {
      accountId: '123456',
      invoiceNumber: '5460026'
    }
  },
  result: {
    data: {
      invoice: {
        invoiceNumber: '5460026',
        status: 'Open',
        terms: 'NET 30 DAYS',
        customerPo: '40251/13325',
        invoiceDate: '10/28/2020',
        dueDate: '10/28/2020',
        originalAmt: '',
        openBalance: '',
        age: '',
        jobNumber: 'AVLNEP7',
        jobName: 'NEP 7 ASHEVILLE',
        address: {
          streetLineOne: '911 BLUEBERRY WAY',
          streetLineTwo: '',
          streetLineThree: '',
          city: 'NORTHLAKE',
          state: 'TX',
          postalCode: '76226',
          country: ''
        },
        shipDate: '11/06/2020',
        deliveryMethod: 'Our Truck',
        subtotal: '3184.28',
        tax: '262.76',
        otherCharges: '0.0',
        paidToDate: '1000.00',
        invoiceItems: [
          {
            id: '1738433F4-91A7-4637-8C60-B4DF59A060C1',
            brand: 'Advanced Drainage Systems, Inc.',
            name: '120 x 7.5, 100 Sq Yd, Geosynthetic, Double, Photodegradable, Erosion Control Blanket',
            partNumber: '8X1125STRAWDN',
            mfr: '00S2TT',
            thumb: 'null',
            price: '43.59',
            qty: {
              quantityOrdered: 1,
              quantityShipped: 1
            }
          }
        ]
      }
    }
  }
};

export const successMincron_5448540 = {
  request: {
    query: GetInvoiceDocument,
    variables: {
      accountId: '91492',
      invoiceNumber: '5448540'
    }
  },
  result: {
    data: {
      invoice: {
        invoiceNumber: '5282411',
        status: 'Open',
        terms: 'NET 30 DAYS',
        customerPo: 'WILKESBORO JIM',
        invoiceDate: '10/06/2021',
        dueDate: '10/28/2020',
        originalAmt: '',
        openBalance: '',
        age: '',
        jobNumber: 'SHOP',
        jobName: 'SHOP',
        address: {
          streetLineOne: '911 BLUEBERRY WAY',
          streetLineTwo: '',
          streetLineThree: '',
          city: 'NORTHLAKE',
          state: 'TX',
          postalCode: '76226',
          country: ''
        },
        shipDate: '11/06/2021',
        deliveryMethod: 'Our Truck',
        subtotal: '2184.28',
        tax: '162.73',
        otherCharges: '0.0',
        paidToDate: '139.91',
        invoiceItems: {
          id: 'F02FD85F-4DCE-4EA7-8459-2DA7CFD689DC',
          brand: 'Certex USA, name:2 x 12, Nylon, Eye and Eye, Ply Sling',
          partNumber: 'SLING212,mfr:CX07-0126-12',
          thumb:
            'https://images.tradeservice.com/ProductImages/DIR100193/CERTEX_CX07-0126-12_LINE_SML.jpg',
          price: '40.00',
          qty: {
            quantityOrdered: 1,
            quantityShipped: 1
          }
        }
      }
    }
  }
};
