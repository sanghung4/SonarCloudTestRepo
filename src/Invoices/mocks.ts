import { subMonths } from 'date-fns';
import { Invoice, InvoicesDocument } from 'generated/graphql';
import { mockRow } from 'test-utils/mockTableInstance';
import { formatDate } from 'utils/dates';

export const success = {
  request: {
    query: InvoicesDocument,
    variables: {
      accountId: '123456',
      erpName: 'ECLIPSE'
    }
  },
  result: {
    data: {
      invoices: {
        bucketNinety: 0.0,
        bucketOneTwenty: 0.0,
        bucketSixty: 0.0,
        bucketThirty: 0.0,
        bucketFuture: 0.0,
        currentAmt: 35224.96,
        invoices: [
          {
            age: 'Future',
            customerPo: '109687',
            invoiceDate: formatDate(subMonths(new Date(), 1)),
            dueDate: '03/01/2022',
            invoiceNumber: 'S110227972.001',
            openBalance: '12.39',
            originalAmt: '12.39',
            status: 'Open',
            invoiceUrl: 'test.com',
            jobName: null,
            jobNumber: null
          },
          {
            age: 'Current',
            customerPo: '1096872',
            invoiceDate: formatDate(subMonths(new Date(), 1)),
            dueDate: '03/01/2022',
            invoiceNumber: 'S110227975.001',
            openBalance: '12.39',
            originalAmt: '12.39',
            status: 'Open',
            invoiceUrl: 'test.com',
            jobName: null,
            jobNumber: null
          },
          {
            age: 'Over120',
            customerPo: '109687',
            invoiceDate: formatDate(subMonths(new Date(), 1)),
            dueDate: '03/01/2022',
            invoiceNumber: 'S110220294.001',
            openBalance: '12.39',
            originalAmt: '12.39',
            status: 'Open',
            invoiceUrl: 'test.com',
            jobName: null,
            jobNumber: null
          },
          {
            age: '91-120',
            customerPo: '109687',
            invoiceDate: '03/01/2021',
            dueDate: '03/01/2022',
            invoiceNumber: 'S110222345.001',
            openBalance: '12.39',
            originalAmt: '12.39',
            status: 'Open',
            invoiceUrl: 'test.com',
            jobName: null,
            jobNumber: null
          },
          {
            age: 'Deposit',
            customerPo: '109687',
            invoiceDate: '03/01/2021',
            dueDate: '03/01/2022',
            invoiceNumber: 'S110220974.001',
            openBalance: '12.39',
            originalAmt: '12.39',
            status: 'Open',
            invoiceUrl: 'test.com',
            jobName: null,
            jobNumber: null
          },
          {
            age: '31-60',
            customerPo: '109687',
            invoiceDate: '03/01/2021',
            dueDate: '03/01/2022',
            invoiceNumber: 'S110227973.001',
            openBalance: '12.39',
            originalAmt: '12.39',
            status: 'Open',
            invoiceUrl: 'test.com',
            jobName: null,
            jobNumber: null
          }
        ],
        totalAmt: 35232.27,
        totalPastDue: 7.3085938
      }
    }
  }
};

export const successMincron = {
  request: {
    query: InvoicesDocument,
    variables: {
      accountId: '123456',
      erpName: 'MINCRON'
    }
  },
  result: {
    data: {
      invoices: {
        bucketNinety: 0.0,
        bucketOneTwenty: 0.0,
        bucketSixty: 0.0,
        bucketThirty: 0.0,
        bucketFuture: 0.0,
        currentAmt: 35224.96,
        invoices: [
          {
            age: 'Future',
            customerPo: '109687',
            invoiceDate: formatDate(subMonths(new Date(), 1)),
            dueDate: '03/01/2022',
            invoiceNumber: 'S110227972.001',
            openBalance: '12.39',
            originalAmt: '12.39',
            status: 'Open',
            invoiceUrl: 'test.com',
            jobNumber: '6566',
            jobName: 'NEP 7 ASHVELLE'
          },
          {
            age: 'Current',
            customerPo: '109687',
            invoiceDate: formatDate(subMonths(new Date(), 1)),
            dueDate: '03/01/2022',
            invoiceNumber: 'S110227975.001',
            openBalance: '12.39',
            originalAmt: '12.39',
            status: 'Open',
            invoiceUrl: 'test.com',
            jobNumber: '56787',
            jobName: 'WALTER HORN'
          },
          {
            age: 'Over120',
            customerPo: '109687',
            invoiceDate: formatDate(subMonths(new Date(), 1)),
            dueDate: '03/01/2022',
            invoiceNumber: 'S110220294.001',
            openBalance: '12.39',
            originalAmt: '12.39',
            status: 'Closed',
            invoiceUrl: 'test.com',
            jobNumber: 'SHOP',
            jobName: 'SEQUAYESH TR'
          },
          {
            age: '91-120',
            customerPo: '109687',
            invoiceDate: formatDate(subMonths(new Date(), 1)),
            dueDate: '03/01/2022',
            invoiceNumber: 'S110222345.001',
            openBalance: '12.39',
            originalAmt: '12.39',
            status: 'Open',
            invoiceUrl: 'test.com',
            jobNumber: 'RETREAT',
            jobName: 'RETREAT AT ARDEN'
          },
          {
            age: 'Deposit',
            customerPo: '109687',
            invoiceDate: formatDate(subMonths(new Date(), 1)),
            dueDate: '03/01/2022',
            invoiceNumber: 'S110220974.001',
            openBalance: '12.39',
            originalAmt: '12.39',
            status: 'Closed',
            invoiceUrl: 'test.com',
            jobNumber: 'AVLNEP7',
            jobName: 'WALTER HORN'
          },
          {
            age: '31-60',
            customerPo: '109687',
            invoiceDate: '03/01/2021',
            dueDate: '03/01/2022',
            invoiceNumber: 'S110227973.001',
            openBalance: '12.39',
            originalAmt: '12.39',
            status: 'Open',
            invoiceUrl: 'test.com',
            jobNumber: 'AIKENON',
            jobName: 'AIKEN AVL'
          }
        ],
        totalAmt: 35232.27,
        totalPastDue: 7.3085938
      }
    }
  }
};

export const mockTestTableMemo = {
  invoices: {
    bucketNinety: 0.0,
    bucketOneTwenty: 0.0,
    bucketSixty: 0.0,
    bucketThirty: 0.0,
    bucketFuture: 0.0,
    currentAmt: 35224.96,
    invoices: [
      {
        age: 'Future',
        customerPo: '109687',
        invoiceDate: '05/02/2022',
        dueDate: '03/01/2022',
        invoiceNumber: 'S110227972.001',
        openBalance: '12.39',
        originalAmt: '12.39',
        status: 'Open',
        invoiceUrl: 'test.com',
        jobNumber: '6566',
        jobName: 'NEP 7 ASHVELLE'
      },
      {
        age: 'Current',
        customerPo: '109687',
        invoiceDate: '03/01/2021',
        dueDate: '03/01/2022',
        invoiceNumber: 'S110227975.001',
        openBalance: '12.39',
        originalAmt: '12.39',
        status: 'Open',
        invoiceUrl: 'test.com',
        jobNumber: '56787',
        jobName: 'WALTER HORN'
      },
      {
        age: 'Over120',
        customerPo: '109687',
        invoiceDate: '03/01/2021',
        dueDate: '03/01/2022',
        invoiceNumber: 'S110220294.001',
        openBalance: '12.39',
        originalAmt: '12.39',
        status: 'Open',
        invoiceUrl: 'test.com',
        jobNumber: 'SHOP',
        jobName: 'SEQUAYESH TR'
      },
      {
        age: '91-120',
        customerPo: '109687',
        invoiceDate: '03/01/2021',
        dueDate: '03/01/2022',
        invoiceNumber: 'S110222345.001',
        openBalance: '12.39',
        originalAmt: '12.39',
        status: 'Open',
        invoiceUrl: 'test.com',
        jobNumber: 'RETREAT',
        jobName: 'RETREAT AT ARDEN'
      },
      {
        age: 'Deposit',
        customerPo: '109687',
        invoiceDate: '03/01/2021',
        dueDate: '03/01/2022',
        invoiceNumber: 'S110220974.001',
        openBalance: '12.39',
        originalAmt: '12.39',
        status: 'Open',
        invoiceUrl: 'test.com',
        jobNumber: 'AVLNEP7',
        jobName: 'WALTER HORN'
      },
      {
        age: '31-60',
        customerPo: '109687',
        invoiceDate: '03/01/2021',
        dueDate: '03/01/2022',
        invoiceNumber: 'S110227973.001',
        openBalance: '12.39',
        originalAmt: '12.39',
        status: 'Open',
        invoiceUrl: 'test.com',
        jobNumber: 'AIKENON',
        jobName: 'AIKEN AVL'
      }
    ],
    totalAmt: 35232.27,
    totalPastDue: 7.3085938
  }
};

const rowToMock = success.result.data.invoices.invoices[0];
export const mockInvoiceRow = mockRow<Invoice>(rowToMock, rowToMock, {});
