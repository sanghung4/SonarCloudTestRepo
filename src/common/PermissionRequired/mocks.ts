import { UserContextType } from 'AuthProvider';
import { Permission } from 'common/PermissionRequired';

export const authConfigNoPerm = {
  authState: {
    isAuthenticated: true
  },
  profile: {
    isEmployee: true,
    isVerified: true
  } as UserContextType['profile'],
  activeFeatures: ['LISTS']
};
export const authConfig = {
  authState: {
    isAuthenticated: true
  },
  profile: {
    userId: 'test',
    permissions: [Permission.EDIT_LIST, Permission.APPROVE_CART],
    isEmployee: true,
    isVerified: true
  } as UserContextType['profile'],
  activeFeatures: ['LISTS']
};
