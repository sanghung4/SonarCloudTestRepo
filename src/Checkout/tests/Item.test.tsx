import ItemDesktop from 'Checkout/Item.desktop';
import ItemMobile from 'Checkout/Item.mobile';
import { lineItemMocks } from 'Order/tests/mocks';
import { render } from 'test-utils/TestWrapper';

describe('Checkout - Item', () => {
  describe('Item.desktop', () => {
    it('should match snapshot with mostly undefined parameters', () => {
      const { container } = render(
        <ItemDesktop lineItem={{}} qtyAvailable={0} />
      );
      expect(container).toMatchSnapshot();
    });

    it('should match snapshot with valid lineItem', () => {
      const { container } = render(
        <ItemDesktop lineItem={lineItemMocks[3]} qtyAvailable={10} index={3} />
      );
      expect(container).toMatchSnapshot();
    });

    it('should match snapshot when disabled', () => {
      const { container } = render(
        <ItemDesktop lineItem={lineItemMocks[1]} qtyAvailable={1} disabled />
      );
      expect(container).toMatchSnapshot();
    });
  });
  describe('Item.mobile', () => {
    it('should match snapshot with mostly undefined parameters', () => {
      const { container } = render(
        <ItemMobile lineItem={{}} qtyAvailable={0} />
      );
      expect(container).toMatchSnapshot();
    });

    it('should match snapshot with valid lineItem', () => {
      const { container } = render(
        <ItemMobile lineItem={lineItemMocks[3]} qtyAvailable={10} index={3} />
      );
      expect(container).toMatchSnapshot();
    });
  });
});
