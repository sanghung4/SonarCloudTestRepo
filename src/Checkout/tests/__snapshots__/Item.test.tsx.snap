// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Checkout - Item Item.desktop should match snapshot when disabled 1`] = `
<div>
  <tr
    class="MuiTableRow-root css-bqkitb-MuiTableRow-root"
    data-testid="checkout-cart-item-null"
  >
    <td
      class="MuiTableCell-root MuiTableCell-sizeMedium css-o1r1tr-MuiTableCell-root"
    >
      <div
        class="MuiBox-root css-70qvj9"
      >
        <div
          class="MuiBox-root css-1mdpkzj"
        >
          <img
            alt="!!! DELIVERY IMMEDIATELY !!!!!"
            class="MuiBox-root css-4w7ia0"
            src=""
          />
        </div>
        <div
          class="MuiBox-root css-1hbndq"
        >
          <span
            class="MuiTypography-root MuiTypography-caption css-rk7jif-MuiTypography-root"
            data-testid="checkout-manufacturer-name-undefined"
          />
          <p
            class="MuiTypography-root MuiTypography-body1 css-4mc1p7-MuiTypography-root"
            data-testid="checkout-product-name-undefined"
          >
            !!! DELIVERY IMMEDIATELY !!!!!
          </p>
          <span
            class="MuiTypography-root MuiTypography-caption css-rk7jif-MuiTypography-root"
            data-testid="checkout-manufacturer-number-undefined"
          >
            MFR#
             
          </span>
        </div>
      </div>
    </td>
    <td
      class="MuiTableCell-root MuiTableCell-alignCenter MuiTableCell-sizeMedium css-1qto78x-MuiTableCell-root"
    >
      <span
        class="MuiTypography-root MuiTypography-caption css-12u2emh-MuiTypography-root"
        data-testid="checkout-unit-price-undefined"
      >
        Price unavailable
      </span>
    </td>
    <td
      class="MuiTableCell-root MuiTableCell-alignCenter MuiTableCell-sizeMedium css-1qto78x-MuiTableCell-root"
    >
      <div
        class="MuiBox-root css-dvxtzn"
      >
        <span
          class="MuiTypography-root MuiTypography-caption css-rk7jif-MuiTypography-root"
          data-testid="checkout-qty-available-undefined"
        >
          1
        </span>
        <span
          class="MuiTypography-root MuiTypography-caption css-xn7l0x-MuiTypography-root"
        >
          in stock
        </span>
      </div>
    </td>
    <td
      class="MuiTableCell-root MuiTableCell-alignCenter MuiTableCell-sizeMedium css-1qto78x-MuiTableCell-root"
    >
      <div
        class="MuiBox-root css-40mi5u"
      >
        <p
          class="MuiTypography-root MuiTypography-body1 css-1vcav5x-MuiTypography-root"
          data-testid="checkout-item-order-qty-undefined"
        >
          0
        </p>
      </div>
    </td>
  </tr>
</div>
`;

exports[`Checkout - Item Item.desktop should match snapshot with mostly undefined parameters 1`] = `
<div>
  <tr
    class="MuiTableRow-root css-bqkitb-MuiTableRow-root"
    data-testid="checkout-cart-item-undefined"
  >
    <td
      class="MuiTableCell-root MuiTableCell-sizeMedium css-1yoxrtx-MuiTableCell-root"
    >
      <div
        class="MuiBox-root css-70qvj9"
      >
        <div
          class="MuiBox-root css-1mdpkzj"
        >
          <img
            alt="Product picture"
            class="MuiBox-root css-4w7ia0"
            src=""
          />
        </div>
        <div
          class="MuiBox-root css-1hbndq"
        >
          <span
            class="MuiTypography-root MuiTypography-caption css-rk7jif-MuiTypography-root"
            data-testid="checkout-manufacturer-name-undefined"
          />
          <p
            class="MuiTypography-root MuiTypography-body1 css-4mc1p7-MuiTypography-root"
            data-testid="checkout-product-name-undefined"
          />
          <span
            class="MuiTypography-root MuiTypography-caption css-rk7jif-MuiTypography-root"
            data-testid="checkout-manufacturer-number-undefined"
          >
            MFR#
             
          </span>
        </div>
      </div>
    </td>
    <td
      class="MuiTableCell-root MuiTableCell-alignCenter MuiTableCell-sizeMedium css-1cubbo7-MuiTableCell-root"
    >
      <span
        class="MuiTypography-root MuiTypography-caption css-12u2emh-MuiTypography-root"
        data-testid="checkout-unit-price-undefined"
      >
        Price unavailable
      </span>
    </td>
    <td
      class="MuiTableCell-root MuiTableCell-alignCenter MuiTableCell-sizeMedium css-1cubbo7-MuiTableCell-root"
    >
      <div
        class="MuiBox-root css-dvxtzn"
      >
        <span
          class="MuiTypography-root MuiTypography-caption css-rk7jif-MuiTypography-root"
          data-testid="checkout-qty-available-undefined"
        >
          0
        </span>
      </div>
    </td>
    <td
      class="MuiTableCell-root MuiTableCell-alignCenter MuiTableCell-sizeMedium css-1cubbo7-MuiTableCell-root"
    >
      <div
        class="MuiBox-root css-40mi5u"
      >
        <p
          class="MuiTypography-root MuiTypography-body1 css-1vcav5x-MuiTypography-root"
          data-testid="checkout-item-order-qty-undefined"
        >
          0
        </p>
      </div>
    </td>
  </tr>
  <tr
    class="MuiTableRow-root css-bqkitb-MuiTableRow-root"
  >
    <td
      class="MuiTableCell-root MuiTableCell-sizeMedium css-11r1h0v-MuiTableCell-root"
      colspan="5"
    >
      <div
        class="MuiBox-root css-10bvlll"
      >
        <svg
          class="MuiBox-root css-fyrd4f"
          sx="[object Object]"
        >
          warning.svg
        </svg>
        <span
          class="MuiTypography-root MuiTypography-caption css-1gbxt6n-MuiTypography-root"
        >
          Item is out of stock or below desired quantity
        </span>
      </div>
    </td>
  </tr>
</div>
`;

exports[`Checkout - Item Item.desktop should match snapshot with valid lineItem 1`] = `
<div>
  <tr
    class="MuiTableRow-root css-bqkitb-MuiTableRow-root"
    data-testid="checkout-cart-item-null"
  >
    <td
      class="MuiTableCell-root MuiTableCell-sizeMedium css-o1r1tr-MuiTableCell-root"
    >
      <div
        class="MuiBox-root css-70qvj9"
      >
        <div
          class="MuiBox-root css-1mdpkzj"
        >
          <img
            alt="PVC 2 DWV 1/8 ST BEND (PDWVS4K) Your # 000755"
            class="MuiBox-root css-4w7ia0"
            src=""
          />
        </div>
        <div
          class="MuiBox-root css-1hbndq"
        >
          <span
            class="MuiTypography-root MuiTypography-caption css-rk7jif-MuiTypography-root"
            data-testid="checkout-manufacturer-name-3"
          />
          <p
            class="MuiTypography-root MuiTypography-body1 css-4mc1p7-MuiTypography-root"
            data-testid="checkout-product-name-3"
          >
            PVC 2 DWV 1/8 ST BEND (PDWVS4K) Your # 000755
          </p>
          <span
            class="MuiTypography-root MuiTypography-caption css-rk7jif-MuiTypography-root"
            data-testid="checkout-manufacturer-number-3"
          >
            MFR#
             
          </span>
        </div>
      </div>
    </td>
    <td
      class="MuiTableCell-root MuiTableCell-alignCenter MuiTableCell-sizeMedium css-1qto78x-MuiTableCell-root"
    >
      <span
        class="MuiTypography-root MuiTypography-caption css-12u2emh-MuiTypography-root"
        data-testid="checkout-unit-price-3"
      >
        $0.82 ea
      </span>
    </td>
    <td
      class="MuiTableCell-root MuiTableCell-alignCenter MuiTableCell-sizeMedium css-1qto78x-MuiTableCell-root"
    >
      <div
        class="MuiBox-root css-dvxtzn"
      >
        <span
          class="MuiTypography-root MuiTypography-caption css-rk7jif-MuiTypography-root"
          data-testid="checkout-qty-available-3"
        >
          10
        </span>
        <span
          class="MuiTypography-root MuiTypography-caption css-xn7l0x-MuiTypography-root"
        >
          in stock
        </span>
      </div>
    </td>
    <td
      class="MuiTableCell-root MuiTableCell-alignCenter MuiTableCell-sizeMedium css-1qto78x-MuiTableCell-root"
    >
      <div
        class="MuiBox-root css-40mi5u"
      >
        <p
          class="MuiTypography-root MuiTypography-body1 css-1vcav5x-MuiTypography-root"
          data-testid="checkout-item-order-qty-3"
        >
          1
        </p>
        <p
          class="MuiTypography-root MuiTypography-body1 css-120qz1h-MuiTypography-root"
          data-testid="checkout-item-total-price-3"
        >
          $0.82
        </p>
      </div>
    </td>
  </tr>
</div>
`;

exports[`Checkout - Item Item.mobile should match snapshot with mostly undefined parameters 1`] = `
<div>
  <div
    class="MuiBox-root css-16686bg"
    data-testid="checkout-cart-item-undefined"
  >
    <div
      class="MuiBox-root css-i3pbo"
    >
      <div
        class="MuiGrid-root MuiGrid-container css-1vam7s3-MuiGrid-root"
      >
        <div
          class="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-6 css-1hhgieh-MuiGrid-root"
        >
          <div
            class="MuiBox-root css-x3cpnl"
          >
            <img
              alt="Product picture"
              class="MuiBox-root css-4w7ia0"
              src=""
            />
          </div>
        </div>
        <div
          class="MuiGrid-root MuiGrid-container MuiGrid-item MuiGrid-grid-xs-6 css-kfrp3-MuiGrid-root"
        >
          <span
            class="MuiTypography-root MuiTypography-body1 css-1nelj9t-MuiTypography-root"
            data-testid="checkout-qty-available-undefined"
          >
            0
          </span>
        </div>
      </div>
    </div>
    <div
      class="MuiGrid-root MuiGrid-container css-11lq3yg-MuiGrid-root"
    >
      <div
        class="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-6 css-1hhgieh-MuiGrid-root"
      >
        <span
          class="MuiTypography-root MuiTypography-caption css-rk7jif-MuiTypography-root"
          data-testid="checkout-manufacturer-name-undefined"
        />
        <p
          class="MuiTypography-root MuiTypography-body1 css-4mc1p7-MuiTypography-root"
          data-testid="checkout-product-name-undefined"
        />
        <span
          class="MuiTypography-root MuiTypography-caption css-rk7jif-MuiTypography-root"
          data-testid="checkout-manufacturer-number-undefined"
        >
          MFR#
           
        </span>
      </div>
      <div
        class="MuiGrid-root MuiGrid-container MuiGrid-item MuiGrid-grid-xs-6 css-187pdxj-MuiGrid-root"
      >
        <div
          class="MuiBox-root css-178yklu"
        >
          <div
            class="MuiBox-root css-pori7h"
          >
            <span
              class="MuiTypography-root MuiTypography-caption css-1xrvs54-MuiTypography-root"
            >
              Order Qty
            </span>
            <div
              class="MuiBox-root css-d0uhtl"
            >
              <span
                class="MuiTypography-root MuiTypography-h5 css-1bvxf5q-MuiTypography-root"
                data-testid="checkout-item-order-qty-undefined"
              >
                0
              </span>
            </div>
          </div>
        </div>
      </div>
    </div>
    <div
      class="MuiGrid-root MuiGrid-container css-12tl881-MuiGrid-root"
    >
      <div
        class="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-6 css-1hhgieh-MuiGrid-root"
      >
        <div
          class="MuiBox-root css-1ebnygn"
        >
          <span
            class="MuiTypography-root MuiTypography-caption css-1xrvs54-MuiTypography-root"
          >
            Price
          </span>
        </div>
        <h5
          class="MuiTypography-root MuiTypography-h5 css-1efrlbf-MuiTypography-root"
          data-testid="checkout-unit-price-undefined"
        >
          $0.00
           
          ea
        </h5>
      </div>
      <div
        class="MuiGrid-root MuiGrid-container MuiGrid-item MuiGrid-direction-xs-column MuiGrid-grid-xs-6 css-kxm2zc-MuiGrid-root"
      >
        <div
          class="MuiBox-root css-1ebnygn"
        >
          <span
            class="MuiTypography-root MuiTypography-caption css-1xrvs54-MuiTypography-root"
          >
            Order Total
          </span>
        </div>
        <h5
          class="MuiTypography-root MuiTypography-h5 css-1efrlbf-MuiTypography-root"
          data-testid="checkout-item-total-price-undefined"
        >
          $0.00
        </h5>
      </div>
    </div>
    <div
      class="MuiBox-root css-l6kq9v"
    >
      <svg
        class="MuiBox-root css-1ay9vb9"
        sx="[object Object]"
      >
        warning.svg
      </svg>
      <span
        class="MuiTypography-root MuiTypography-caption css-1gbxt6n-MuiTypography-root"
      >
        Item is out of stock or below desired quantity
      </span>
    </div>
  </div>
  <div
    class="MuiBox-root css-ibnnmb"
  >
    <hr
      class="MuiDivider-root MuiDivider-fullWidth css-di86mu-MuiDivider-root"
    />
  </div>
</div>
`;

exports[`Checkout - Item Item.mobile should match snapshot with valid lineItem 1`] = `
<div>
  <div
    class="MuiBox-root css-16686bg"
    data-testid="checkout-cart-item-null"
  >
    <div
      class="MuiBox-root css-i3pbo"
    >
      <div
        class="MuiGrid-root MuiGrid-container css-1vam7s3-MuiGrid-root"
      >
        <div
          class="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-6 css-1hhgieh-MuiGrid-root"
        >
          <div
            class="MuiBox-root css-x3cpnl"
          >
            <img
              alt="PVC 2 DWV 1/8 ST BEND (PDWVS4K) Your # 000755"
              class="MuiBox-root css-4w7ia0"
              src=""
            />
          </div>
        </div>
        <div
          class="MuiGrid-root MuiGrid-container MuiGrid-item MuiGrid-grid-xs-6 css-kfrp3-MuiGrid-root"
        >
          <span
            class="MuiTypography-root MuiTypography-body1 css-1nelj9t-MuiTypography-root"
            data-testid="checkout-qty-available-3"
          >
            10
          </span>
          <p
            class="MuiTypography-root MuiTypography-body1 css-6apigl-MuiTypography-root"
          >
            in stock
          </p>
        </div>
      </div>
    </div>
    <div
      class="MuiGrid-root MuiGrid-container css-11lq3yg-MuiGrid-root"
    >
      <div
        class="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-6 css-1hhgieh-MuiGrid-root"
      >
        <span
          class="MuiTypography-root MuiTypography-caption css-rk7jif-MuiTypography-root"
          data-testid="checkout-manufacturer-name-3"
        />
        <p
          class="MuiTypography-root MuiTypography-body1 css-4mc1p7-MuiTypography-root"
          data-testid="checkout-product-name-3"
        >
          PVC 2 DWV 1/8 ST BEND (PDWVS4K) Your # 000755
        </p>
        <span
          class="MuiTypography-root MuiTypography-caption css-rk7jif-MuiTypography-root"
          data-testid="checkout-manufacturer-number-3"
        >
          MFR#
           
        </span>
      </div>
      <div
        class="MuiGrid-root MuiGrid-container MuiGrid-item MuiGrid-grid-xs-6 css-187pdxj-MuiGrid-root"
      >
        <div
          class="MuiBox-root css-178yklu"
        >
          <div
            class="MuiBox-root css-pori7h"
          >
            <span
              class="MuiTypography-root MuiTypography-caption css-1xrvs54-MuiTypography-root"
            >
              Order Qty
            </span>
            <div
              class="MuiBox-root css-d0uhtl"
            >
              <span
                class="MuiTypography-root MuiTypography-h5 css-1bvxf5q-MuiTypography-root"
                data-testid="checkout-item-order-qty-3"
              >
                1
              </span>
            </div>
          </div>
        </div>
      </div>
    </div>
    <div
      class="MuiGrid-root MuiGrid-container css-12tl881-MuiGrid-root"
    >
      <div
        class="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-6 css-1hhgieh-MuiGrid-root"
      >
        <div
          class="MuiBox-root css-1ebnygn"
        >
          <span
            class="MuiTypography-root MuiTypography-caption css-1xrvs54-MuiTypography-root"
          >
            Price
          </span>
        </div>
        <h5
          class="MuiTypography-root MuiTypography-h5 css-1efrlbf-MuiTypography-root"
          data-testid="checkout-unit-price-3"
        >
          $0.82
           
          ea
        </h5>
      </div>
      <div
        class="MuiGrid-root MuiGrid-container MuiGrid-item MuiGrid-direction-xs-column MuiGrid-grid-xs-6 css-kxm2zc-MuiGrid-root"
      >
        <div
          class="MuiBox-root css-1ebnygn"
        >
          <span
            class="MuiTypography-root MuiTypography-caption css-1xrvs54-MuiTypography-root"
          >
            Order Total
          </span>
        </div>
        <h5
          class="MuiTypography-root MuiTypography-h5 css-1efrlbf-MuiTypography-root"
          data-testid="checkout-item-total-price-3"
        >
          $0.82
        </h5>
      </div>
    </div>
  </div>
  <div
    class="MuiBox-root css-ibnnmb"
  >
    <hr
      class="MuiDivider-root MuiDivider-fullWidth css-di86mu-MuiDivider-root"
    />
  </div>
</div>
`;
